lib/shopify_api/auth/jwt_payload.rb

```diff
@@ -27,15 +27,15 @@ def initialize(token)
           decode_token(token, T.must(Context.old_api_secret_key))
         end
 
-        @iss = T.let(payload_hash["iss"], String)
+        @iss = T.let(payload_hash["iss"], T.nilable(String))
         @dest = T.let(payload_hash["dest"], String)
         @aud = T.let(payload_hash["aud"], String)
         @sub = T.let(payload_hash["sub"], String)
```

Comment:
In my testing I found that the `sub` claim could also be nill, if the customer is not logged in.